assets/tests/integration/rec-with-table/variable-details-sheet.csv

```diff
@@ -0,0 +1,4 @@
+variable,dummyVariable,typeEnd,databaseStart,variableStart,typeStart,recEnd,numValidCat,catLabel,catLabelLong,units,recStart,catStartLabel,variableStartShortLabel,variableStartLabel,notes
+var_one,,cat,db_one,[var_one_start],cat,copy,2,,,,else,,,,
+var_two,,cat,db_one,DerivedVar::[var_one],cat,Func::var_two_func,2,,,,N/A,,,,
+var_three,,cat,db_one,[var_two],cat,copy,2,,,,else,,,,
```

Comment:
@yulric Out of curious, are users expected to fill `databaseStart` as `db_one` for the last two rows? Neither `var_two` nor `var_three` is derived from a variable that exists in `db_one` (unless you recursively go back). I'm wondering what the "proper" convention is here since this scenario wasn't discussed in the vignettes.

Reply:
The way the code currently works, users are required to specify the `databaseStart` column for derived variables. I think it makes sense to do that to be very clear about what databases is valid for. Saying that, the code can probably figure out the databseStart value for a derived variable from the start variable for it.